src/clib/xtg/grd3d_ecl_tsteps.c

```diff
@@ -80,7 +80,10 @@ grd3d_ecl_tsteps(FILE *fc, int *seqnums, int *day, int *mon, int *year, int nmax
     tofree = keywords;
     ic = 0;
     nc = 0;
-    //    while ((token = strsep(&keywords, "|")) != NULL) {
+
+    // with LGR's, multiple INTEHEAD may occurs; need here to ensure first
+    // INTEHEAD after SEQNUM
+    int look_for_intehead = 0;
```

Comment:
Should be `bool` instead?

Reply:
Will try (see if Windows accepts this C99 construct)